docs/usage.md

```diff
@@ -115,6 +115,10 @@ This file contains several information that can be obtained as gtdb-tk output. Y
 
 This file contains several information that can be obtained as checkM output. You can either use directly the same file found there or you can make a custom one.
 
+### Check duplicates
+
+The pipeline is checking if there are any duplicates among the genomes that the user provides. If there are duplicates, the pipeline will stop and return a file with the contigs that needs to be changed in their name in order to work. This is done to avoid overlapping in the following steps (e.g. same prokka output for the protein sequences and the gffs).
```

Comment:
```suggestion
The pipeline is checking if there are any duplicate names among the genomes that the user provides. If there are duplicates, the pipeline will stop and return a file with the contig names that needs to be changed in their name in order to work. This is done to avoid overlapping in the following steps (e.g. same prokka output for the protein sequences and the gffs).
```